definitions/divorce/json/CaseField/CaseField-nonprod.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "LiveFrom": "23/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "HelpWithFeesReferenceNumber",
+    "Label": "Help with fees reference",
+    "FieldType": "Text",
+    "RegularExpression": "([Hh][Ww][Ff]-?)?[0-9a-zA-Z]{3}-?[0-9a-zA-Z]{3}$",
```

Comment:
can you please add unit tests for this regex?

Reply:
@yarpo I used the same definition that exists for D8HelpWithFeesReferenceNumber and SolicitorFeeAccountNumber as these new fields are doing same job.